extensions/amp-bind/0.1/test/test-bind-impl.js

```diff
@@ -58,62 +42,108 @@ describes.realWin('Bind', {
       newElement.className = 'i-amphtml-foo -amp-foo amp-foo';
       newElement.mutatedAttributesCallback = () => {};
     }
-    const parent = env.win.document.getElementById('parent');
     parent.appendChild(newElement);
     return newElement;
-  }
+  };
+}
 
+function onBindReady_(bind, env) {
   /**
-   * Resolves when Bind service is fully initialized.
    * @return {!Promise}
    */
-  function onBindReady() {
+  return function() {
     return bind.initializePromiseForTesting().then(() => {
       env.flushVsync();
     });
-  }
+  };
+}
 
+function onBindReadyAndSetState_(bind, env) {
```

Comment:
Hmm, maybe. I was trying not to cause a lot of changes in this class but I'll probably follow up with a clean-up PR as mentioned in the TODOs.